packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -234,6 +234,12 @@ class ScrollPhysics {
   /// scroll position to fulfill such a request.
   bool get allowImplicitScrolling => true;
 
+  /// Generates a semantic scroll value, or null if none should be created.
+  String semanticScrollValue({
```

Comment:
Is it intentional that this is not overwritten anywhere with an actual implementation? Where is this one used?